package.json

```diff
@@ -13,15 +13,15 @@
     "build": "cd website && ../node_modules/.bin/docusaurus-build && cp _redirects build/all-contributors",
     "write-translations": "cd website && ../node_modules/.bin/docusaurus-write-translations",
     "crowdin": "java -jar website/crowdin/crowdin-cli.jar",
-    "crowdin-upload": "yarn crowdin --config crowdin.yaml upload sources --auto-update -b $(git rev-parse --abbrev-ref HEAD)",
-    "crowdin-download": "yarn crowdin --config crowdin.yaml download -b $(git rev-parse --abbrev-ref HEAD)",
-    "build-with-translations": "yarn write-translations && yarn crowdin-upload && yarn crowdin-download && yarn build",
+    "crowdin-upload": "npm run crowdin -- --config crowdin.yaml upload sources --auto-update -b $(git rev-parse --abbrev-ref HEAD)",
```

Comment:
@JimMadge i've modified the scripts to ensure npm (and npx) is used throughout. If you have time to test on your end that would be wonderful!